ethereum/core/src/main/java/org/hyperledger/besu/ethereum/core/BlockHeader.java

```diff
@@ -332,6 +332,7 @@ public static org.hyperledger.besu.ethereum.core.BlockHeader convertPluginBlockH
         blockHeaderFunctions);
   }
 
+  @Override
   public String toLogString() {
     return getNumber() + " (" + getHash() + ")";
```

Comment:
It might be unnecessary. I only included timestamp in the other one because it didn't have access to getHash and thought something was better than nothing. In reality, this impl will be used instead of the ProcessableBlockHeader impl, and I'm not sure what value adding timestamp would bring above the hash.